src/compliance/compliance-requirements.controller.ts

```diff
@@ -137,4 +139,125 @@ export class ComplianceRequirementsController {
     const result = await this.complianceRequirementsService.unpauseRequirements(ticker, params);
     return handleServiceResult(result);
   }
+
+  @ApiOperation({
+    summary: 'Delete single compliance requirement for an Asset',
+    description: 'This endpoint removes referenced compliance requirement for an Asset.',
+  })
+  @ApiParam({
+    name: 'ticker',
+    description:
+      'The ticker of the Asset from which the compliance requirements is to be removed from',
+    type: 'string',
+    example: 'TICKER',
+  })
+  @ApiParam({
+    name: 'id',
+    description: 'The id of the compliance requirement to be deleted',
+    type: 'string',
+    example: '123',
+  })
+  @ApiTransactionResponse({
+    description: 'Details of the transaction',
+    type: TransactionQueueModel,
+  })
+  @ApiTransactionFailedResponse({
+    description: 'The Asset or compliance requirement was not found',
+  })
+  @Delete(':id')
```

Comment:
```suggestion
  @Post(':id/remove')
```

The project name is a bit of a misnomer, at least in regards to how modification is performed. We follow REST API design except for any writing operation we write `POST resource/action`, at least for chain actions.

 Block chains are typically append only data structures, so every write is creating new things. We tried at first to be more strict on RESTFUL conventions, but it was a bit awkward in spots, especially when the chain API is RPC style.